### PR TITLE
flutter music player UI

### PR DESCRIPTION
**Pull request by Kavindu-Gayantha**
related toi issue  #1 .

### CODE REVIEW

**Comment by Safnaj** [comment]
Hi @Kavindu-Gayantha, The UI Looks Cool.

I request to do the following changes to merge the PR.

- [ ] Change music_app to First Letter Capital as Music_App
- [ ] The UI in readme file looks very big
- [ ] Also update a Mockup same as in the Main ReadMe file

**Review by Safnaj — CHANGES_REQUESTED** [review]
Changes Requested

**Comment by Kavindu-Gayantha** [comment]
> Hi @Kavindu-Gayantha, The UI Looks Cool.
> 
> I request to do the following changes to merge the PR.
> 
>     * [ ]  Change music_app to First Letter Capital as Music_App
> 
>     * [ ]  The UI in readme file looks very big
> 
>     * [ ]  Also update a Mockup same as in the Main ReadMe file

Ok , I will make changes.

**Comment by Kavindu-Gayantha** [comment]
@Safnaj All done. 
* [x]  Change music_app to First Letter Capital as Music_App

* [x]  The UI in readme file looks very big

* [x]  Also update a Mockup same as in the Main ReadMe file

File: music_app/android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
          FlutterApplication and put your custom class here. -->
     <application
         android:name="io.flutter.app.FlutterApplication"
-        android:label="music_app"
+        android:label="Music_App"
```
Comment (Safnaj) [inline music_app/android/app/src/main/AndroidManifest.xml]: Also change the Main Project name as Music_App

**Review by Safnaj — CHANGES_REQUESTED** [review]

File: music_app/android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Safnaj) [reply music_app/android/app/src/main/AndroidManifest.xml]: Also add ur mockup in the main readme file here

[ReadMe Link](https://github.com/Safnaj/Flutter-UI-Samples/blob/master/README.md)

Comment (Kavindu-Gayantha) [reply music_app/android/app/src/main/AndroidManifest.xml]: ok. got it

**Review by Safnaj — APPROVED** [review]

**Comment by Safnaj** [comment]
Hi @Kavindu-Gayantha, I have added hactoberfest-accepted label for you. In order to merge ur PR, pls change the project name as first capital letter. Check the below image.

Screenshot 2020-10-23 at 12 43 17 AM

**Comment by Kavindu-Gayantha** [comment]
> Hi @Kavindu-Gayantha, I have added **hactoberfest-accepted** label for you. In order to merge ur PR, pls change the project name as first capital letter. Check the below image.
> 
> Screenshot 2020-10-23 at 12 43 17 AM

previously i changed name in android manifest file also. I thought it solved. now changed in the packages. as Music_App

**Comment by Safnaj** [comment]
@Kavindu-Gayantha i was telling to change this not the manifest file. As ler the unpublished code of conduct we follow a proper naming convention. Therefore it has to be a capital letter

**Comment by Kavindu-Gayantha** [comment]
> @Kavindu-Gayantha i was telling to change this not the manifest file. As ler the unpublished code of conduct we follow a proper naming convention. Therefore it has to be a capital letter

I thought it might have solved like that, I renamed now again.

**Pull request merged**